Frontend/package.json

```diff
@@ -31,7 +31,7 @@
     "babel-jest": "25.5.1",
     "eslint": "6.8.0",
     "jest": "25.5.4",
-    "metro-react-native-babel-preset": "0.59.0",
+    "metro-react-native-babel-preset": "0.51.0",
```

Comment:
React native update to 0.64 and metro downgraded? Is this an intended change, does it break anything?